Sources/Application/Environment/Configuration/Debug.xcconfig

```diff
@@ -5,17 +5,12 @@ PRODUCT_BUNDLE_IDENTIFIER = uk.gov.onelogin.build
 INFOPLIST_KEY_CFBundleDisplayName = One Login - Build
 INFOPLIST_KEY_NSCameraUsageDescription = This app does not use the camera
 
-BASE_URL = mobile.build.account.gov.uk
-EXTERNAL_BASE_URL = signin.account.gov.uk
+BASE_URL = account.gov.uk
+STS_BASE_URL = token.build.$BASE_URL
+MOBILE_BE_BASE_URL = mobile.build.$BASE_URL
 
-FEATURE_FLAG_FILE = FeatureFlagsBuild
-
-ONE_LOGIN_AUTHORIZE_URL = auth-stub.mobile.build.account.gov.uk
-ONE_LOGIN_CLIENT_ID = TEST_CLIENT_ID
-
-REDIRECT_URL = https:/$()/mobile.build.account.gov.uk/redirect
-STS_BASE_URL = token.build.account.gov.uk
 STS_CLIENT_ID = bYrcuRVvnylvEgYSSbBjwXzHrwJ
 
-WALLET_CREDENTIAL_ISSUER_URL = https:/$()/example-credential-issuer.mobile.build.account.gov.uk
```

Comment:
Do we not need the issuer URL anymore?

Reply:
It's passed into the `WalletSDK.start` method on the current wallet version we're using 🤷‍♂️

Reply:
So it's still needed?

Reply:
For our current version of wallet (5.0.0) it is I think, yes 🙂